cytominer_eval/operations/percent_strong.py

```diff
@@ -38,13 +39,17 @@ def percent_strong(
     # Check to make sure that the melted dataframe is upper triangle
     assert_melt(similarity_melted_df, eval_metric="percent_strong")
 
+    # check that there ARE group_replicates (non-unique rows)
+    replicate_df = similarity_melted_df.query("group_replicate")
+    denom = replicate_df.shape[0]
+
+    ### HYT's addition, though i dont know if this will work
+    assert denom != 0, "no replicate groups identified!"
```

Comment:
```suggestion
    assert denom != 0, "no replicate groups identified in {rep} columns!".format(rep=replicate_groups)
```

I like to add more informative error messages - doing this will provide quicker feedback to the user.

Also, if you accept this commit, please also do run `black` again (i.e. I didn't check for style).